src/app/modal/modal-form.demo.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component} from "@angular/core";
+
+export class Hero {
+    constructor(public id: number, public name: string, public power: string, public alterEgo?: string) {}
+}
+
+@Component({
+    moduleId: module.id,
+    selector: "clr-modal-dynamic-form",
+    styleUrls: ["./modal.demo.css"],
+    templateUrl: "./modal-form.demo.html"
+})
+export class ModalFormDemo {
+    basic: boolean = false;
+    powers = ["Really Smart", "Super Flexible", "Super Hot", "Weather Changer"];
+
+    model = new Hero(18, "Dr IQ", this.powers[0], "Chuck Overstreet");
+
+    submitted = false;
+
+    onSubmit(form: any) {
+        this.submitted = true;
+        console.log(form);
+    }
+
+    // TODO: Remove this when we're done
```

Comment:
Are we done so this should be removed?

Reply:
That was from the demo code I copied in to develop with. Will remove the comment.